cfme/automate/explorer/method.py

```diff
@@ -228,6 +229,37 @@ def read(self):
         return self.all_vars
 
 
+class EntryPoint(Text, ClickableMixin):
+    def fill(self, value):
+        """Used to select the automate method from automate domain"""
+        if value:
+            self.click()
+            self.parent_view.embedded_view.wait_displayed('15s')
```

Comment:
This is making a big assumption about the parent view for this widget.  From the structure you have here, are these `EntryPoint` widgets displayed _on_ the `EmbeddedView` modal?  If so, they should be a widget defined within `EmbeddedView`.

You can use `ParametrizedView` and `ParametrizedLocator` to be able to define multiple `EmbeddedView` instances with different locators for `EntryPoint` that's displayed on them.

Reply:
I am removing this widget as we have common widget ```EntryPoint```